exercises/practice/crypto-square/.meta/proof.ci.wren

```diff
@@ -0,0 +1,57 @@
+class CryptoSquare {
+  static columns(n) {
+    var c = 0
+    while (c * c < n) {
+      c = c + 1
+    }
+    return c
```

Comment:
```suggestion
    c
```

Most of the return statements here can be removed, last expression is returned by default.

Reply:
I have removed the `return` keyword in single-line functions.

In other contexts, I observe `null` being returned when the `return` keyword is omitted.

In existing `proof.ci.wren` files, only the following never use `return` :
```
exercises/practice/high-scores/.meta/proof.ci.wren
exercises/practice/leap/.meta/proof.ci.wren
exercises/practice/resistor-color/.meta/proof.ci.wren
exercises/practice/robot-name/.meta/proof.ci.wren
```